src/paths/Home/Home.js

```diff
@@ -12,6 +14,11 @@ const Home = ({main, track}) => {
 
   return (
     <div className="Home">
+      <SEO
+          title={`Amhrán #${track.order}: ${track.artistName} - ${track.songName}`}
+          url={window.location.href}
+          image={`${window.location.href}images/${track.image}`}
```

Comment:
The issue with this is that `window.location.href` isn't necessarily the homepage. It could be `*homepage*/004`, etc.

Reply:
Putting the images in /public/ sadly doesn't play nicely with the redirect set up in the app. If you navigate to *localhost*/public/001.png, it redirects you to a 404. I can look into that.